tests/acceptance/books-test.js

```diff
@@ -25,4 +25,18 @@ module('Acceptance | books', function(hooks) {
     assert.ok(page.books(0).hasAuthor);
     assert.ok(page.books(0).hasImage);
   });
+
+  test('fetch more data when scrolled down', async function(assert) {
+    await page.visit();
+
+    assert.equal(page.books().count, 9);
+
+    document.querySelector(page.loaderClass).scrollIntoView();
+
+    await waitUntil(() => {
+      return page.books().count === 18;
+    });
+
+    assert.equal(page.books().count, 18);
+  });
```

Comment:
The tests seem to use the actual api, meaning that they will hit the api every time the test runs.

Can you think of why this might not be a good idea in real live app and what are some alternatives?